src/style.scss

```diff
@@ -57,16 +58,12 @@ $auro-inset-directions:'--squish';
   }
 }
 
-// handle focus-visible state
-:host(.focus-visible) {
-  .auro-button {
-    @include focus-visible;
+:focus-visible.auro-button {
+  @include focus-visible;
+}
 
-    &::-moz-focus-inner {
-      border: none;
-      width: 100%;
-    }
-  }
+.focus-visible.auro-button {
```

Comment:
Same comment as above here about chained selectors.

Reply:
I don't think we can combine these -- the focus-visible class is added by the [focus-visible polyfill](https://github.com/WICG/focus-visible), and is not controlled by us.